index.html

```diff
@@ -99,6 +99,30 @@
           issues on GitHub (see links below). All comments are welcome.
         </p>
       </div>
+      <section>
+        <h2>
+          Features at risk
+        </h2>
+        <p>
+          As this specification enters the Candidate Recommendation phase of
+          the W3C standardization process, the working group has identified the
+          following feature(s) as being "<dfn>at risk</dfn>" of being removed
+          from the specification. The working group seeks input from
+          implementers, developers, and the general public on whether these
+          features should remain in the specification. If no compelling use
+          cases are received, these features will be removed from the
```

Comment:
> or if there is insufficient implementation experience

It's not about experience, it's about implementer interest. Updated text to reflect that.